tools/depends/target/crossguid/Makefile

```diff
@@ -52,7 +52,7 @@ endif
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
 
 $(LIBDYLIB): $(PLATFORM)
-	cd $(PLATFORM); $(CXX) -c guid.cpp -o guid.o $(CXXFLAGS) -D$(GUID_PLATFORM_DEFINE)
+	cd $(PLATFORM); $(CXX) -std=c++11 -c guid.cpp -o guid.o $(CXXFLAGS) -D$(GUID_PLATFORM_DEFINE)
```

Comment:
does `CXXFLAGS` not contain `-std=c++11`?

Reply:
Presumably not for me as build passes with this patch and fails without.

Reply:
This really should not be needed. 
please check if you Makefile.include has CXXFLAGS=-std=c++11

Reply:
I suspect this is again a case of native compiler used for the C++11 check...
Do you use a very recent _native_ gcc that supports c++11 by default?

Reply:
Ubuntu 17.10
$ gcc --version
gcc (Ubuntu 7.2.0-8ubuntu3) 7.2.0
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Reply:
For reference, I was plagued with this error on Android for months and @koying came to the rescue: #12974

Reply:
the real fix would be to refactor the c++11 detection.
In the meantime, please add CXXFLAGS += -std=c++11 after L12 instead of here